interface/slack.py

```diff
@@ -13,42 +13,42 @@ def __init__(self, sc):
 
     def send_dm(self, message, slack_user_id):
         """Send direct message to user with id of slack_user_id."""
-        logging.debug("Sending direct message to {}".format(slack_user_id))
+        logging.debug(f"Sending direct message to {slack_user_id}")
         response = self.sc.api_call(
             "chat.postMessage",
             channel=slack_user_id,
             text=message
         )
         if 'ok' not in response:
-            logging.error("Direct message to {} failed with error: {}".
-                          format(slack_user_id, response['error']))
+            logging.error(f"Direct message to {slack_user_id} failed with "
+                          f"error: {response['error']}")
             raise SlackAPIError(response['error'])
 
     def send_to_channel(self, message, channel_name, attachments=[]):
         """Send message to channel with name channel_name."""
-        logging.debug("Sending message to channel {}".format(channel_name))
+        logging.debug(f"Sending message to channel {channel_name}")
         response = self.sc.api_call(
             "chat.postMessage",
             channel=channel_name,
             attachments=attachments,
             text=message
         )
         if 'ok' not in response:
-            logging.error("Message to channel {} failed with error: {}".
-                          format(channel_name, response['error']))
+            logging.error(f"Message to channel {channel_name} failed with "
+                          f"error: { response['error']}")
```

Comment:
Space between left bracket and `r` where there is no space between the right bracket: consistency

Reply:
fixed